src/windows.rs

```diff
@@ -1,24 +1,39 @@
-use winapi::shared::minwindef;
-
 use std::os::windows::io::AsRawHandle;
 use std::process::Child;
-use std::ptr;
+
+mod windows {
+    use core::ffi::c_void;
+    pub(crate) use windows::Win32::{
+        Foundation::HANDLE,
+        System::{
+            Diagnostics::Debug::{ReadProcessMemory, WriteProcessMemory},
+            Threading::{
+                OpenProcess, PROCESS_CREATE_THREAD, PROCESS_QUERY_INFORMATION,
+                PROCESS_VM_OPERATION, PROCESS_VM_READ, PROCESS_VM_WRITE,
+            },
+        },
+    };
+
+    pub(crate) type DWORD = u32;
```

Comment:
Does the `windows` crate not define these anywhere? Also clippy will require that you explicitly allow these types to be all caps.

Reply:
The windows crate does not define these types. Instead they directly use the native datatype, for which DTYPE etc are just typedefs. I defined these type aliases according to [the api docs](https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types) to make the transition from the winapi crate easier. I can get rid of them completely if you want. Otherwise I will annotate them for clippy.

Reply:
Following the reasoning from [this comment](https://github.com/Tommoa/rs-process-memory/pull/11#discussion_r1022263012), I removed the type aliases for LPCVOID and LPVOID for now. I can reintroduce them if desired.